gslab_scons/provenance.py

```diff
@@ -41,11 +40,11 @@ def make_provenance(start_path, readme_path, provenance_path,
                               find_for_me, start_path, verbose)
     except Exception as e:
         print('make_provenance.py failed.')
```

Comment:
See https://github.com/gslab-econ/gslab_python/pull/120#discussion_r136410044. This line isn't nice to read, and it points too a non-existant script.

Reply:
meant to write `make_provenance()`

Reply:
Is a user of `release.py` guaranteed to know what `make_provenance()` is/does?

Reply:
`release.py` has its own README and guide which we will update. I treat `release.py` as an advanced functionality, not something new users will touch much. That's how we currently treat it in template as well

Reply:
Great point! We should document these optional CL args in that readme. Want to do it as part of this task, or just note it in gslab-econ/template#82. I'd vote for this task.